### PR TITLE
feat: complete support for badger api-v2

### PR DESCRIPTION
**Pull request by NamelsKing**

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -1,23 +1,18 @@
...
 } from './types';
-import {
-  Account,
-  APIOptions,
-  LeaderboardSummary,
-  ProtocolMetrics,
-  ProtocolSummary,
-  Vault,
-} from './interfaces';
+import * as i from './interfaces';
```
Comment (axejintao) [inline src/api/index.ts:10]: is there a reason to do it this way vs. modular imports? not sure if i like this lower case i but also you don't see things like this in java

Comment (NamelsKing) [reply src/api/index.ts:10]: it`s not the rule, just i`m doing * all the time i have imports > 5. To keep file readability. We can keep it your way for consistency

**Review by axejintao — APPROVED** [review]

**Pull request merged**